### DISCUSSION TITLE
Time to calibrate

### DISCUSSION THREAD

**Discussion by doobes** [eoyilmaz/displaycal-py3#315] [Q&A]
So,

I've done a flathub install on my Elementary OS box.

Connected my Spyder5Pro and started up DisplayCal.

Going through the process, I'm now "Measuring color swatches for calibration" and I'm 53 minutes into this and it's on Patch 88 of 175.

How long should the calibration process take?

This seems a bit long.

Thanks,

**Comment by eoyilmaz** [comment]
It depends on the device and the monitor, my i1 Display Pro for 175 patches on my Dell UP2516D takes around 9-10 minutes but the same patches on my Mac Book Pro takes 15-16 minutes (with XYZ LUT + Matrix, which is not supported in MacOS tbh) because the Mac Book Pro's monitor has much deeper blacks.
